packages/core/types/src/README.md

```diff
@@ -1,37 +0,0 @@
-# Strapi Types Documentation
-
-### @strapi/types/core
-
-This package encapsulates core Strapi types. The types within are foundational elements of Strapi's server-side functionalities. They majorly represent services, middlewares, main objects and core application operational entities.
-
-### @strapi/types/modules
-
-The 'modules' package signifies Strapi's modular design principle. It contains types that characterize the Strapi's modules, maintaining separation of concerns and promoting modularity. As these modules tend to be functional components, their types often dictate structure and behavior.
-
-### @strapi/types/utils
-
-The 'utils' package is a utility belt for types in Strapi. The types in this package aid in bolstering type safety, code readability and functional expressiveness of the codebase. They don't necessarily map directly to business entities but serve to supplement them.
```

Comment:
I can't see this per folder description captured anywhere else, do you think it's a good idea to remove it?

Reply:
Good question, this references the package structure I wanted to have with custom exports and not directly the folders (even though it could def work for them too).

Since we're not continuing with the custom exports map, I thought that we might delete that & add back this information in a later task regarding the "concepts".

We could also have a small readme.md per directory to explain what it contains.

What do you think?

Reply:
Yep that sounds good to me 👍🏻